Sources/Paywalls/Components/PaywallV2CacheWarming.swift

```diff
@@ -37,14 +37,19 @@ extension PaywallComponentsData {
 extension PaywallComponentsData.PaywallComponentsConfig {
 
     var allImageURLs: [URL] {
```

Comment:
I'm taking over this PR for Josh while he's out of the country. I wanted to get your feedback on this…

it says ALL image urls. But currently we're only returning the non-heic images.

Josh put in some logic to only override that for the sheet component so that the HEIC images are prefetched too. 

Any reason why we don't just return ALL the image urls? Including the HEIC?

@facumenzella @MarkVillacampa @fire-at-will